arch/sim/Kconfig

```diff
@@ -159,6 +159,20 @@ config SIM_HOSTFS
 	---help---
 		Access host filesystem through HostFS.
 
+if SIM_HOSTFS
+config SIM_IMAGEPATH_AS_CWD
+	bool "Simulator switch working directory"
```

Comment:
as this approach affects more than hostfs, i guess it's better to make this option independent from hostfs.
how do you think?

Reply:
At present I think its impact is limited to hostfs, because nuttx does not request other path-related data from host.

If it needs to be changed, I haven't figured out where to put the "host_fs_init" function. If it is placed in sim_hostfs.c, it needs to rely on SIM_HOSTFS

Reply:
done
I put it inside sim_misc.c so it can be independent of hostfs

Reply:
thank you.
for example, `SIM_NETDEV_VPNKIT_PATH` can be affected.